source/tutorials/moving-pictures.rst

```diff
@@ -234,7 +234,7 @@ The FastTree program creates an unrooted tree, so in the final step in this sect
 Alpha and beta diversity analysis
 ---------------------------------
 
-QIIME 2's diversity analyses are available through the ``q2-diversity`` plugin, which supports computing alpha and beta diversity metrics, applying related statistical tests, and generating interactive visualizations. We'll first apply the ``core-metrics-phylogenetic`` method, which rarefies a ``FeatureTable[Frequency]`` to a user-specified depth, and then computes a series of alpha and beta diversity metrics. The metrics computed by default are:
+QIIME 2's diversity analyses are available through the ``q2-diversity`` plugin, which supports computing alpha and beta diversity metrics, applying related statistical tests, and generating interactive visualizations. We'll first apply the ``core-metrics-phylogenetic`` method, which rarefies a ``FeatureTable[Frequency]`` to a user-specified depth, and then computes a series of alpha and beta diversity metrics as well as some exploratory Emperor plots. The metrics computed by default are:
```

Comment:
_and then computes a series of alpha and beta diversity metrics as well as some exploratory Emperor plots_ 

to 

_computes several alpha and beta diversity metrics, and generates principal coordinates analysis (PCoA) plots using Emperor for each of the beta diversity metrics._